code/game/objects/structures/signs.dm

```diff
@@ -148,4 +148,48 @@
 /obj/structure/sign/directions/evac
 	name = "escape arm"
 	desc = "A direction sign, pointing out which way the escape shuttle dock is."
-	icon_state = "direction_evac"
\ No newline at end of file
+	icon_state = "direction_evac"
+
+/obj/structure/sign/safetyboard //64 by 32 pixels, takes two tiles
+	name = "Station Safety Sign"
+	desc = "Only you can prevent station spaghettification."
+	icon = 'icons/obj/safetyboards.dmi'
+	icon_state = "major_accident"
+	var/value
+	var/record
+	var/record_desc = " The gold trim signifies that we've reached the current record for safety; no pressure!"
+
+/obj/structure/sign/safetyboard/New()
+	while(ticker.current_state < GAME_STATE_PLAYING)

```

Comment:
if you move this code to object initialize (called AFTER the map and ticker have loaded by the object controller) you can save this while loop.

Then all you have to do is make it detect when ones are created midround by admins.